bedrock/mozorg/templates/mozorg/fxa-iframeless-test.html

```diff
@@ -0,0 +1,34 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import fxa_email_form with context %}
+
+{% extends "mozorg/base-resp.html" %}
+
+{% block page_title %}{{ _('Get to know Mozilla') }}{% endblock %}
+{% block body_id %}about{% endblock %}
+{% block body_class %}sand{% endblock %}
+{% block body_attrs %}{{ super() }} data-global-nav-current-link="about-us"{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('fxa_iframeless_test') }}
+{% endblock %}
+
+{% block content %}
+<main>
+  <div id="fxaccounts-wrapper">
+    <div id="fxa-form-wrapper">
+      {# Bug 1354710: firstrun pages need to pass funnelcake parameters to FxA #}
+      {% set utm_source = 'firstrun' if not funnelcake_id else 'firstrun_f' + funnelcake_id %}
+      {{ fxa_email_form(entrypoint='firstrun', utm_source=utm_source, utm_params={'campaign': 'fxa-embedded-form', 'content': 'fx-{}'.format(version), 'medium': 'referral'}) }}
```

Comment:
The utm_content value is `fx-` in every browser I tried, not sure what the intention was.

Reply:
Ah, yes, good catch. This was a straight copy from the old firstrun pages, which were passed `version` in the template context, which contained the version of firefox being run for the first time (e.g. the `61.0.1` in the URL). As this is just a demo page, we can ignore here. (Real/different`utm_content` values will ostensibly be used in real implementations.)

Reply:
👍